### PR TITLE
Type patches

### PR DESCRIPTION
**Pull request by asze17**
To whomever it might concern,
    Such PR is created by me, myself for the sole purpose of fixing Tom the Bomb (@Tom-the-Bomb)'s types, such said, it shalt be important to state the fact that this PR is provided "AS IS", without warranty of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose and noninfringement. in no event shall the authors or copyright holders be liable for any claim, damages or other liability, whether in an action of contract, tort or otherwise, arising from, out of or in connection with the software or the use or other dealings in the software.<br/>
    According to @jay3332, he stated "OMG TOM UNFRIEND ME SO SUSPISHOUS", such quote reveals that tom the bomb is rather unfriendly towards others, which is why his code formatting is extremely concerning yet after numerous reminders by me and jay he still yet resist to change. Another evidence clear shows that Tom the Bomb lacks the ability to write proper code, "i think i just confused bomb by saying attribute!/even though that's what they are called!" (Jay3332 @jay3332). Such quote clearly expresses that Tom the Bomb lacks the ability to understand the Rust Programming Language, which is vital for writing safe and efficient code, which Tom the Bomb clearly lacks. Another quote that shows Tom the Bomb lacks the skill of communication is "instead let's try retrieving bomb last name" (Jay3332 @jay3332) Tom the Bomb still yet refuses to reveal his last name, his true identity, this clearly shows that Tom the Bomb lacks the skill of communication, which is why he refuses to improve his code style, which is related to his personal growth and wellness.<br/>
    Such evidences allows me to conclude that Tom the Bomb, shalt merge this PR as a result of his lack of knowledge regarding the Rust Programming Language and his lack of communication skill.<br/>
    Also format your code with black, it is really concerning.

### CODE REVIEW

**Comment by jay3332** [comment]
Alvin S, your wellness has been immediately demanded

File: discord_games/wordle.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 from __future__  import annotations
-from typing import Optional
 
 import pathlib
 import random
 import asyncio
+from typing import Optional, Final
```
Comment (jay3332) [inline discord_games/wordle.py:6]: Alvin S, that is the wrong place to import typing. Additionally consider ordering thy imports alphabetically.

Comment (asze17) [reply discord_games/wordle.py:6]: Jay3332 I know such but I do not want to format the entire thing for Tom the bomb

**Comment by Tom-the-Bomb** [comment]
I request the removal of the `TYPE_CHECKING` block

**Pull request merged**